src/app/main/component/events/services/events-comments.service.ts

```diff
@@ -51,11 +51,11 @@ export class EventsCommentsService implements CommentsService {
     return this.http.get<number>(`${this.backEnd}events/comments/likes/count/commentId=${id}`);
   }
 
-  getRepliesAmount(id: number): Observable<number> {
+  getRepliesAmount(entityId: number, id: number): Observable<number> {
```

Comment:
`entityId` is too generic and doesn't indicate what it represents. It would be clearer as commentId or eventId  or newId  or userId based on its actual usage.